ci-operator/config/openshift-priv/installer/openshift-priv-installer-main.yaml

```diff
@@ -588,14 +588,17 @@ tests:
     workflow: openshift-e2e-gcp
   timeout: 6h0m0s
 - always_run: false
-  as: e2e-gcp-custom-dns
+  as: gcp-custom-dns
   optional: true
+  run_if_changed: (gcp|google)
   steps:
     cluster_profile: gcp-qe
     env:
       FEATURE_GATES: '["GCPClusterHostedDNSInstall=true"]'
       FEATURE_SET: CustomNoUpgrade
       USER_PROVISIONED_DNS: "yes"
+    test:
+    - chain: cucushift-installer-check-cluster-health
```

Comment:
So this will stop e2e tests from running on the custom DNS job? 

Are we doing this because the e2e tests are permafailing? I am ok with that, but not sure if there are other options? 

@sadasu & I were discussing this today and there were roughly ~10 tests that failed on both of the runs for the custom DNS job. I believe @gpei mentioned this before, and that it would not make sense to add ignores for each individual test.

Reply:
> because the e2e tests are permafailing

This is one of the reasons. Some others:
1. We set a public domain which resides in QE project, so it depends on QE's cluster profile. While the QE cloud project have a small number of budget. So skip the e2e testing to save the running time for saving the cloud cost.
2. The job is for installer presubmits jobs, most of time, we are focus on installation results, so I drop the e2e to avoid the noise from unexpected e2e failure.

Reply:
The most ideal solution in practice would be to fix the tests in the origin repo, either by making them pass or skipping them, so that we can see the most accurate and reasonable e2e test results.
But I believe that having each team update their test code to adapt all test cases to every new cluster type we introduced would be a big workload for them, and potentially with a very lengthy timeline.
During our initial testing of custom DNS, I informed the Ingress team that certain tests were consistently failing. They submitted PR https://github.com/openshift/origin/pull/29985 to fix these issues, but the PR is still under review to this day.

Reply:
@jianlinliu and @gpei thanks for that explanation. I agree that installer pre-submit jobs should can be limited to verifying a successful install. 

We will be able to catch other errors during the periodic e2e custom-dns jobs. I am concerned about scenarios like openshift/origin#29985  and how that will affect the health of our periodic job within CR.